charts/amazon-cloudwatch-observability/templates/operator-deployment.yaml

```diff
@@ -32,6 +32,7 @@ spec:
         - "--auto-instrumentation-python-image={{ template "auto-instrumentation-python.image" . }}"
         - "--auto-instrumentation-dotnet-image={{ template "auto-instrumentation-dotnet.image" . }}"
         - "--auto-instrumentation-nodejs-image={{ template "auto-instrumentation-nodejs.image" . }}"
+        - "--target-allocator-image={{ template "target-allocator.modify-image" }}"
```

Comment:
don't we need `imageDomain` like other images for `target-allocator.modify-image` in template helper?

Reply:
I believe it's present, but in its own block below.

https://github.com/aws-observability/helm-charts/blob/73382ddb0d7626973fd032c1356026ab8b667774/charts/amazon-cloudwatch-observability/templates/_helpers.tpl#L153-L159